ui/src/components/startpageDemo/Startpage.js

```diff
@@ -4,7 +4,7 @@ import './startpage.css';
 import httpRequests from '../../helpers/httpRequests';
 import InspectExamples from '../inspectExamplesDemo/InspectExamples';
 import Dropzone from "../uploadERC/Dropzone/Dropzone";
-import config from '../../helpers/config';
+//import config from '../../helpers/config';
```

Comment:
If something is not needed anymore, please remove, or add a further comment why this might be needed in the future.